README.md

```diff
@@ -33,6 +33,25 @@ curl http://localhost:9003/writetagged -s -X POST -d '{"namespace":"metrics","id
 curl http://localhost:9003/query -s -X POST -d '{"namespace":"metrics","query":{"regexp":{"field":"city","regexp":".*"}},"rangeStart":0,"rangeEnd":'"$(date +"%s")"'}' | jq .
 ```
 
+## Building with Docker
+
+A Dockerfile is included for both development and production deployment purposes. It uses a
```

Comment:
maybe add a caveat that the config needs to be updated based on what environment you're running on.